helpers/notifications.js

```diff
@@ -1,13 +1,13 @@
 const { ObjectId } = require("mongodb");
 const getMongoConnection = require("./mongoConnect");
 const { Expo } = require("expo-server-sdk");
-const cron = require("node-cron");
 
 const attachPushToken = async (req, res, next) => {
   try {
     const decodedTokenData = req.body.userID;
     const pushToken = req.body.expoPushToken;
     const mongoConnection = await getMongoConnection();
+    const decodedTokenData = req.tokenData;
```

Comment:
@SeanP1tt 

1. decodedTokenData is already declared on line 7. Since you are planning to pull user ID from the JWT, you can safely delete line 7
2. the decodedTokenData looks like this: 
```
{
  data: { _id: '5ec18a6744dece00247f3acf', email: 'anandangalig@gmail.com' },
  iat: 1593446370,
  exp: 1601222370
}
```
So you want **req.tokenData.data._id** to get the user ID
3. The middleware verifyAndAttachJWTData needs to be added to **app.post("/push", verifyAndAttachJWTData, attachPushToken);** to extract and attach decoded JWT data so your **attachPushToken** can access it in the **req** object.

Reply:
Let me know if any of these are un-clear. We can jump on a screenshare and I can help you navigate things.